src/GravityFormsExtensions.php

```diff
@@ -278,11 +278,11 @@ function ( $conf ) use ( $confirmation ) {
 		$confirmation_fields = [
 			'confirmation'    => $default_confirmation['message'],
 			'share_platforms' => [
-				'facebook' => '1' === $default_confirmation['facebook'] ?? false,
-				'twitter'  => '1' === $default_confirmation['twitter'] ?? false,
-				'whatsapp' => '1' === $default_confirmation['whatsapp'] ?? false,
-				'native'   => '1' === $default_confirmation['native'] ?? false,
-				'email'    => '1' === $default_confirmation['email'] ?? false,
+				'facebook' => isset( $default_confirmation['facebook'] ) ? '1' === $default_confirmation['facebook'] : true,
```

Comment:
I have one comments and one comment about this,

1. What is supposed to returns each `$default_confirmation[...]`? `0` or `1`?
2. Because all this social media have applied the same logic, you can move it to a function, like:
```
function fnName (value): boolean {
 return isset(value) ? '1' === value : true
}
```
And then 
```
'facebook' => fnName( $default_confirmation['facebook'] ),
'twitter' => fnName( $default_confirmation['twitter'] ),
...
```
What do you think?

Reply:
Added a [commit](https://github.com/greenpeace/planet4-master-theme/pull/1800/commits/eea944b0ee45bb8a0aab31586f296ae545b16a30) with the suggested refactoring!

Reply:
I actually went with [Sagar's suggested refactoring](https://github.com/greenpeace/planet4-master-theme/pull/1800#discussion_r977767326) since it simplifies the code quite a bit and seems to work just as well 🙂